client.go

```diff
@@ -53,6 +54,12 @@ type Client struct {
 // Timeout
 //
 // By default the HTTP client doesn't sets a timeout. See the Timeout(time.Duration) method for more detail.
+//
+// UserAgent
+//
+// Please set a UserAgent HTTP Header with the name of the app/service querying, the version, system and a URL if
+// you have one, so that you are not blocked by mistake
+// (e.g. CoolFoodApp - Go - Version 1.0 - https://coolfoodapp.com)
```

Comment:
perhaps give it a sensible, default value in `NewClient`?

Reply:
I'm not sure what default value should we use.

Reply:
What about 'OpenFoodFacts - Go - v0.0 - https://github.com/openfoodfacts/openfoodfacts-go' ?

Reply:
Yes this one looks good. Changes are pushed